language/move-lang/src/hlir/translate.rs

```diff
@@ -1468,7 +1473,7 @@ fn check_trailing_unit(context: &mut Context, block: &mut Block) {
             let semi_msg = "Invalid trailing ';'";
             let unreachable_msg = "Any code after this expression will not be reached";
             let info_msg = "A trailing ';' in an expression block implicitly adds a '()' value \
-            after the semicolon. That '()' value will not be reachable";
+                        after the semicolon. That '()' value will not be reachable";
```

Comment:
Shouldn't this align with the opening quote?

Reply:
No idea, its what cargo fmt did with `format_strings=true`

Reply:
Weird. It seems to have done something different here than in other cases. Oh well.